app/javascript/react/components/BrandShowContainer.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react'
+
+const BrandShowContainer = (props) => {
+    const [pizzas, setPizzas] = useState([])
+    const [brand, setBrand] = useState({name: ''})
+
+    let brandId = props.match.params.id
+
+    const getPizzas = async () => {
+	    try {
+	      const response = await fetch(`/api/v1/brands/${brandId}`)
+	      if (!response){
+	        const errorMessage = `${response.status} (${response.statusTest})`
+	        const error = new Error(errorMessage)
+	        throw(error)
+	      }
+	      const parsedPizzas= await response.json()
+	      setPizzas(parsedPizzas.pizzas)
+        setBrand(parsedPizzas.brand)
+	    } catch(err){
+	      console.error(`Error in fetch: ${err.message}`)
+	    }
+    }  
+    
```

Comment:
tabs are off here. Make sure you update your VS code to have the same settings with 2 tabs.